src/test/java/racingCar/CarTest.java

```diff
@@ -0,0 +1,23 @@
+package racingCar;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import racingCar.strategy.TestMovingStrategy;
+import racingcar.domain.Car;
+import racingcar.strategy.MovingStrategy;
+
+public class CarTest {
+
+  @Test
+  @DisplayName("2회 이동 시 위치값이 2")
+  void testcarPostion() {
+    Car car = new Car("car1");
+    MovingStrategy movingStrategy = new TestMovingStrategy();
+    car.move(movingStrategy);
```

Comment:
테스트 👍 
테스트용 전략 클래스를 만들 수도 있지만, 아래와 같이 람다를 사용할 수도 있습니다 😄 
```suggestion
    car.move(() -> true);
```